package.json

```diff
@@ -62,12 +62,12 @@
     "react-icons": "^4.11.0",
     "react-indiana-drag-scroll": "^2.2.0",
     "react-markdown": "^9.0.0",
+    "rehype-prism-plus": "^1.6.3",
```

Comment:
This is another instance of a package we are including in the library now because it needs to be a production dependency for Next.js. Is there a way to tell Vercel to install `devDependencies` when it builds the website? That would help us get around needing to have these as production dependencies, which are included for every user who runs `npm i flowbite-react` (until monorepo)

Also, I haven't actually tried it but in this instance, you may well be able to install this as a development dependency and it works just fine, since the original rehype plugin did. Should probably try that first.

Reply:
I think we should not worry about extra dependencies ending up on users since the monorepo is just around the corner, which invalidates all this at times.

Reply:
I will go through every single thing once the monorepo work is started and I think that's the best time to sort all packages installed and put them right where they should be (dep, or devDep)

Reply:
Yeah I just wanted to keep that on your radar. It's already annoying people (rightly so) and will continue to do so

Reply:
Totally agreed!